src/ontology/cl-edit.owl

```diff
@@ -27034,12 +27034,11 @@ SubClassOf(obo:CL_1000721 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000516
 
 # Class: obo:CL_1000742 (glomerular mesangial cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "Wikipedia:Intraglomerular_mesangial_cell") obo:IAO_0000115 obo:CL_1000742 "Intraglomerular mesangial cells are specialized pericytes located among the glomerular capillaries within a renal corpuscle of a kidney.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://doi.org/10.1152/physrev.1998.78.3.723>) Annotation(oboInOwl:hasDbXref "Wikipedia:Intraglomerular_mesangial_cell") obo:IAO_0000115 obo:CL_1000742 "A specialized pericyte located among the glomerular capillaries in a renal corpuscle.")
```

Comment:
While there is a large number of publications [considering mesangial cells a type of pericyte](https://pubmed.ncbi.nlm.nih.gov/3308611/), there are others that [argue they have same origin but they are different](https://www.frontiersin.org/articles/10.3389/fmed.2021.740527/full#:~:text=MCs%20are%20considered,cells%20(8)). In [this publication](https://www.nature.com/articles/s41581-021-00474-8), they show that they have different markers, localisation and functions:
![image](https://github.com/obophenotype/cell-ontology/assets/94959119/172225fb-e1c0-4741-8a66-492bcd37725c)

Consider changing the definition for: "A mesangial cell located among the glomerular capillaries in a renal corpuscle.".
Of note, `mesangial cell` is currently a SubclassOf `pericyte`.